src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSFabricDetails.cs

```diff
@@ -864,6 +868,25 @@ public ASRHyperVReplicaAzureSpecificRPIDetails(HyperVReplicaAzureReplicationDeta
         /// </summary>
         public string RecoveryProximityPlacementGroupId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the SQL Server license type to the machine to in the event of a failover.
```

Comment:
>machine to [](start = 60, length = 10)

Please correct the description, something like
Gets or sets the SQL Server license type of the machine in the event of a failover.

Reply:
Done

Reply:
Done

---
In reply to: [601542526](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601542526) [](ancestors = 601542526)